index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h2>Task 1</h2>
+    
+    <p>Here are some inline elements in HTML:</p>
+
+    <ul>
```

Comment:
empty line can by between large section ? or for example ol, ul. 
but not between single tags, it's not practical .